ui/app/api/mirrors/state/route.ts

```diff
@@ -1,17 +1,15 @@
-import {
-  MirrorStatusRequest,
-  MirrorStatusResponse,
-} from '@/grpc_generated/route';
+import { MirrorStatusResponse } from '@/grpc_generated/route';
 import { GetFlowHttpAddressFromEnv } from '@/rpc/http';
 
 export async function POST(request: Request) {
-  const body: MirrorStatusRequest = await request.json();
+  const body = await request.json();
+
   const flowServiceAddr = GetFlowHttpAddressFromEnv();
   try {
     const res: MirrorStatusResponse = await fetch(
```

Comment:
```suggestion
    return fetch(
```
See how columns lookup optimized in https://github.com/PeerDB-io/peerdb/pull/1877